release_notes/43-release-notes.adoc

```diff
@@ -327,6 +328,26 @@ It also provides the following security fixes:
 * Helm: Missing YAML content leads to panic link:https://access.redhat.com/security/cve/cve-2024-26147[(CVE-2024-26147)]
 * Helm: Shows secrets with `--dry-run` option in clear text link:https://access.redhat.com/security/cve/cve-2019-25210[(CVE-2019-25210)]
 
+[id="resolved-in-version-437_{context}"]
+=== Resolved in version 4.3.7
+
+*Release date*: 8 May 2024
+
+This release provides the following bug fixes:
+
+* This release fixes an issue where the Central pod failed with a `SQLSTATE 23503` error after upgrading {product-title-short} to release 4.3.6. This resulted in the collector pods showing the `CrashLoopBackoff` state because the collector probe was unavailable.
+* This release updates the Scanner baseline vulnerability data to address changes made to the Red Hat security data feeds that were not compatible with earlier data from Scanner's scheduled feed processing. This fixes various issues where vulnerabilities were detected for images containing packages that were previously incorrectly indicated as affected by a vulnerability.
+* This release fixes a crash and rendering error in the network graph that occurs when Central is running a {product-title-short} release of 4.3.6 or earlier and Sensor is running a {product-title-short} release of 4.4.0 or later.
```

Comment:
Is RHACS pronounced by spelling the letters out? If so, I think that means the vowel sound of "R" requires a different article here:
```suggestion
* This release fixes a crash and rendering error in the network graph that occurs when Central is running an {product-title-short} release of 4.3.6 or earlier and Sensor is running an {product-title-short} release of 4.4.0 or later.
```